cloud-init/add-https-certificate-block-to-cs-profiles.tpl.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Generate a random value for the password.
+password=$(openssl rand -base64 32)
+
+# Convert the full chain and private key PEMs into a single P12 file.
+#
+# This is a Terraform template file, and the domain, full_chain_pem,
+# and priv_key_pem variables are passed in via templatefile().
+#
+# shellcheck disable=SC2154
+openssl pkcs12 -export -in "${full_chain_pem}" -inkey "${priv_key_pem}" \
+  -out "/tmp/${domain}.p12" -name "${domain}" -passout pass:"$password"
+
+# Create the Java keystore from the P12 file.
+#
+# TODO: issue #121 suggests improving this code by allowing the user
+# to specify the location of the Java keystore.
```

Comment:
I would like where the issue is included at a minimum just so it's specifically mentioned. These issues aren't always in the repos they're mentioned so I think it's better to be explicit.

```suggestion
# TODO: issue cisagov/cool-assessment-terraform#121 suggests improving
# this code by allowing the user to specify the location of the Java
# keystore.
```